web/src/app/services/ServiceDetails.js

```diff
@@ -42,123 +53,96 @@ const query = gql`
-      if (!data.alerts.nodes || !data.alerts.nodes.length) {
-        alertStatus = 'ok'
-      } else if (data.alerts.nodes[0].status === 'StatusUnacknowledged') {
-        alertStatus = 'err'
-      } else {
-        alertStatus = 'warn'
-      }
-    }
-
-    let titleFooter = null
-    if (data.service.ep) {
-      titleFooter = (
-        <div>
-          Escalation Policy:&nbsp;
-          <Link to={`/escalation-policies/${data.service.ep.id}`}>
-            {data.service.ep.name}
-          </Link>
-        </div>
-      )
-    }
-    return (
-      <React.Fragment>
-        <PageActions>
-          <QuerySetFavoriteButton serviceID={data.service.id} />
-          <OtherActions
-            actions={[
-              {
-                label: 'Edit Service',
-                onClick: () => this.setState({ edit: true }),
-              },
-              {
-                label: 'Delete Service',
-                onClick: () => this.setState({ delete: true }),
-              },
-            ]}
-          />
-        </PageActions>
-        <DetailsPage
-          title={data.service.name}
-          details={data.service.description}
-          titleFooter={titleFooter}
-          links={[
-            {
-              label: 'Alerts',
-              status: alertStatus,
-              url: 'alerts',
-            },
-            {
-              label: 'Heartbeat Monitors',
-              url: 'heartbeat-monitors',
-              status: hbStatus(data.service.heartbeatMonitors),
-            },
+  return (
+    <React.Fragment>
+      <PageActions>
+        <QuerySetFavoriteButton serviceID={serviceID} />
+        <OtherActions
+          actions={[
             {
-              label: 'Integration Keys',
-              url: 'integration-keys',
+              label: 'Edit Service',
+              onClick: () => setShowEdit(true),
             },
             {
-              label: 'Labels',
-              url: 'labels',
+              label: 'Delete Service',
+              onClick: () => setShowDelete(true),
             },
           ]}
-          pageFooter={<ServiceOnCallQuery serviceID={this.props.serviceID} />}
         />
-        {this.state.edit && (
-          <ServiceEditDialog
-            onClose={() => this.setState({ edit: false })}
-            serviceID={this.props.serviceID}
-          />
-        )}
-        {this.state.delete && (
-          <ServiceDeleteDialog
-            onClose={() => this.setState({ delete: false })}
-            serviceID={this.props.serviceID}
-          />
-        )}
-      </React.Fragment>
-    )
-  }
-
-  render() {
-    return (
-      <Query
-        query={query}
-        partialQuery={titleQuery}
-        variables={{ serviceID: this.props.serviceID }}
-        render={this.renderData}
+      </PageActions>
+      <DetailsPage
+        title={data.service.name}
+        details={data.service.description}
+        titleFooter={
+          <div>
+            Escalation Policy:&nbsp;
```

Comment:
Let's decide on using `&nbsp;` or `{' '}`, since we have both

Reply:
I hadn't noticed, might be a bug. Does `{' '}` translate to a non-breaking space even if it's a normal (non-breaking) space?

Reply:
I believe it adds to the DOM the same way as `&nbsp;`. I think the linter adds `{' '}` when it tries to break a long string in JSX, so using that might be easier to maintain in the long run, unless we change that rule

Reply:
I think in this instance it may not even matter (since `&nbsp` is just so multiple spaces don't get squished into one). I think `{' '}` will do wherever we just need a single space.

If we need multiple, I think `&nbsp;` is clearer since `{' '}` and `{' '}` look exactly the same otherwise.

Reply:
It doesn't really matter which way we go with, just wanted to pick one since both are present

Reply:
Updated to use `{' '}` for regular spaces